caluma_interval/client.py

```diff
@@ -58,7 +58,11 @@ def execute(self, query, variables=None):
 
     def _send(self, query, variables):
         data = {"query": query, "variables": variables}
-        headers = {"Accept": "application/json", "Content-Type": "application/json"}
+        headers = {
+            "Accept": "application/json",
+            "Content-Type": "application/json",
+            "Caluma-Companion": "interval",
```

Comment:
As said above, I'd prefer a preshared secret instead of just a "guessable" value